### PR TITLE
A warm tea recipe

### PR DESCRIPTION
**Pull request by eafyounian**
This is a cool recipe to warm your day!

### CODE REVIEW

**Comment by sofiarandelin** [comment]
WOW

**Comment by sofiarandelin** [comment]
Nice warm tea

File: tea.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+- take the teabag
+- boil some water
```
Comment (ftabaro) [inline tea.txt:2]: I like my tea frozen cold!

**Comment by joonatansipola** [comment]
How long should you keep the teabang in the water?

**Comment by eafyounian** [comment]
> How long should you keep the teabag in the water?

Good point! I will update the recipe!

File: tea.txt (hunk 1 above)
Comment (joonatansipola) [reply tea.txt:2]: You should make a separate iced tea recipe

**Comment by joonatansipola** [comment]
Awesome! I see nothing to improve. Simple and exquisite.

**Pull request merged**